core/app/models/spree/credit_card.rb

```diff
@@ -152,9 +152,9 @@ def require_card_numbers?
 
     def ensure_one_default
       if self.user_id && self.default
-        CreditCard.where(default: true).where.not(id: self.id).where(user_id: self.user_id).each do |ucc|
-          ucc.update_columns(default: false)
-        end
+        CreditCard.where(default: true, user_id: self.user_id)
+          .where.not(id: self.id)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Redundant `self` detected.